_posts/2016-01-10-release-connector4java-1.8.md

```diff
@@ -0,0 +1,175 @@
+
+To delete a client:
+
+{% highlight java %}
+connector.deleteClient("short-living-client", accessToken);
+{% endhighlight %}
+
+To change a client:
+
+{% highlight java %}
+Client client = new Client.Builder("example-client", "new-secret")
+        .accessTokenValiditySeconds(300)
+        .refreshTokenValiditySeconds(60000)
+        .redirectUri("http://localhost:5055/oauth2")
+        .scopes(Sets.newHashSet(Scope.ME.getValue()))
+        .grants(Sets.newHashSet(GrantType.REFRESH_TOKEN.name(),
+                                GrantType.AUTHORIZATION_CODE.name()))
+        .implicit(true)
+        .validityInSeconds(120)
+        .build();
+
+Client updatedClient = connector.updateClient("example-client", client, accessToken);
+{% endhighlight %}
+
+Note that `updatedClient` might not contain the same values as `client`.
+The server can override given values and will set defaults for missing values.
+
+## Network Timeouts per Connector
+
+You can now set network timeouts per connector instance:
+
+{% highlight java %}
+connector = new OsiamConnector.Builder()
+        ...
+        .withConnectTimeout(150)
+        .withReadTimeout(500)
+        ...
+        .build();
+{% endhighlight %}
+
+You can still globally set timeouts like before.
+Use this to set timeouts for all connector instances or global defaults that can be overridden per connector.
+For more information see the [Connector4Java documentation](https://github.com/osiam/connector4java/blob/v1.8/docs/create-osiam-connector.md#timeouts).
+
+## Compatibility
+
+In release 1.7 the Connector4Java introduced an incompatibility with versions of OSIAM <= 2.3.
+This release restore this compatibility.
+To connect to OSIAM <= 2.3, you have to set the configuration option `withLegacySchemas` on the `OsiamConnector` to `true`:
+
+{% highlight java %}
+OsiamConnector osiamConnector = new OsiamConnector.Builder()
+       ...
+       .withLegacySchemas(true)
+       ...
+       .build();
+{% endhighlight %}
+
+For more information see the [Connector4Java documentation](https://github.com/osiam/connector4java/blob/v1.8/docs/create-osiam-connector.md#legacy-schemas).
+
+Also, support for OSIAM 3.0 has been added.
+This enables you to test your current applications against OSIAM 3.0, that is still in development, without having to change the dependencies.
+To connect to OSIAM 3.0, you have to use the configuration option `withEndpoint` on the `OsiamConnector` instead of the other endpoint methods:
+
+{% highlight java %}
+OsiamConnector osiamConnector = new OsiamConnector.Builder()
+       ...
+       .withEndpoint(OSIAM_ENDPOINT)
+       ...
+       .build();
+{% endhighlight %}
+
+For more information see the [Connector4Java documentation](https://github.com/osiam/connector4java/blob/v1.8/docs/create-osiam-connector.md#osiam-3x).
+
+## Immutable SCIM Classes
+
+We always thought, that all SCIM classes were immutable.
+But taking a closer look revealed, that this wasn't the case.
```

Comment:
here as well. Also get rid of the but or decapitalize it and turn the period into a comma